content/bn/docs/reference/glossary/cla.md

```diff
@@ -0,0 +1,18 @@
+---
+title: সিএলএ (কন্ট্রিবিউটর লাইসেন্স চুক্তি)
+id: cla
+date: 2018-04-12
+full_link: https://github.com/kubernetes/community/blob/master/CLA.md
+short_description: >
+  যেসকল শর্তাবলীর জন্য ওপেন সোর্স প্রজেক্টে অনুদানকারীদের একটি লাইসেন্স অনুমোদন করা হয়।
+
+aka: 
+tags:
+- community
+---
+যেসকল শর্তাবলীর জন্য ওপেন সোর্স প্রজেক্টে {{< glossary_tooltip text="অনুদানকারী" term_id="contributor" >}} 
+একটি লাইসেন্স অনুমোদন করা হয়।
```

Comment:
একজন {{< glossary_tooltip text="অবদানকারী" term_id="contributor" >}} তার অবদানের জন্য ওপেন সোর্স প্রকল্পকে যে সকল শর্তের অধীনে লাইসেন্স প্রদান করেন।

Reply:
I have fixed the Texts....